modules/cpmso-yaml-provider-spec-aws.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control-plane-machine-management.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-provider-spec-aws_{context}"]
+= Sample AWS provider specification
+
+The provider specification must match that of the control plane machines created by the installation program. You can omit any field that is set in the failure domain section of the CR.
+// "that of the control plane machines" -> i.e., in their `machine` resource? We should explain how to reference this regardless.
```

Comment:
Yes, the providerSpec in the machine resource will be largely similar to the CPMS providerSpec. We compare the two to determine whether a machine needs to be replaced/updated